roles/postgres/handlers/main.yml

```
@@ -0,0 +1,5 @@
+- name: Restart postgres.
+  service:
+    name: postgresql
```

This should use `postgresql_daemon` to select the proper cluster.

```suggestion
    name: '{{ postgresql_daemon }}'
```